.github/workflows/ci.yml

```
@@ -199,6 +199,50 @@ jobs:
         command: test
         args: --no-run --test codegen beat_codegen
 
+  coverage:
```

Just to be consistent with the naming convention of other jobs.

```suggestion
  coverage:
    name: Coverage
```